### PR TITLE
Make all workloads sticky with no remove option

### PR DESCRIPTION
**Pull request by karelhala**
animation

### CODE REVIEW

**Comment by karelhala** [comment]
ping @terezanovotna

**Comment by terezanovotna** [comment]
Nice work 👏👏

File: src/js/App/GlobalFilter/constants.js [hunk 1]
```diff
@@ -15,6 +16,14 @@ export const workloads = [
     }
 ];
 
+export const selectWorkloads = () => ({
+    [workloads?.[0]?.tags?.[0]?.tag?.key]: {
```
Comment (Hyperkid123) [inline src/js/App/GlobalFilter/constants.js:20]: So it is acceptable to have an `undefined` key? Should we have some fallback like `default` or something?

Comment (karelhala) [reply src/js/App/GlobalFilter/constants.js:20]: Good point! I'll add `All workloads` as default.

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**